### ISSUE TITLE
LocalJumpError: no block given (yield) - When using `run_step` method

### ISSUE THREAD

**Issue by navinmuthu** [allure-framework/allure-ruby#58]
We are currently using allure-spec (0.8.0) in our project and found that this version got deprecated. So, we are planning to upgrade allure to 2.13. Now, we are facing issue in `run_step` method.

Sample Code:
```
it "Sample code to check run_step method" do |t|
  t.run_step("Step 1") do
     p "print statement 1"
  end
end
```
Output:
```
LocalJumpError:
   no block given (yield)
```
Stack Trace:
```
# /Users/navinmuthu/.rvm/gems/ruby-2.5.1@falcon_allure_2.13.1/gems/allure-ruby-commons-2.13.1/lib/allure-ruby-commons.rb:156:in `run_step'
# /Users/navinmuthu/.rvm/gems/ruby-2.5.1@falcon_allure_2.13.1/gems/allure-rspec-2.13.1/lib/allure_rspec/formatter.rb:24:in `block (3 levels) in <class:RSpecFormatter>'
```
Please let me know how to use `run_step` method.

**Comment by navinmuthu** [comment]
On debugging this issue, we see that block is not handled in below code block present in `allure-rspec-2.13.1/lib/allure_rspec/formatter.rb`
```
RSpec::Core::Example.class_eval do
   Allure.singleton_methods.each do |method|        
     define_method(method) { |*args| Allure.__send__(method, *args) }
  end
end
```
After modifying like below, run_step works fine. (Handling block in define_method)
```
RSpec::Core::Example.class_eval do
   Allure.singleton_methods.each do |method|        
     define_method(method, &block) { |*args| Allure.__send__(method, *args, &block) }
  end
end
```

**Comment by andrcuns** [comment]
@navinmuthu Thanks for the issue. I am aware of this issue, It was on my list, just didn't yet get to fixing it yet. Feel free to create PR if You like or I will do it sometime this week most likely.

**Comment by navinmuthu** [comment]
Thanks @andrcuns for quick reply.

Looks like I don't have access to push changes to remote branch :(

**Comment by andrcuns** [comment]
@navinmuthu  Of course You don't, You need to fork it, do the changes and create PR from Your own repo :)

**Comment by andrcuns** [comment]
Closed via #59
